tests/rt/is_realtime.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright Â© 2009-2013 Bernhard M. Wiedemann
+# Copyright Â© 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+# Check that realtime kernel is running
+sub run() {
+    select_console 'root-console';
+    script_run 'systemctl mask packagekit.service';
+    script_run 'systemctl stop packagekit.service';
```

Comment:
is there a reason you need to do this here? isn't that taken care of by consoletest_setup?